test/services/cache/ExportsDeclarationBuilder.scala

```diff
@@ -227,7 +227,7 @@ trait ExportsDeclarationBuilder {
     meansOfTransportCrossingTheBorderNationality: Option[String] = None,
     container: Boolean = false,
     meansOfTransportCrossingTheBorderType: String = "",
-    meansOfTransportCrossingTheBorderIDNumber: Option[String] = None,
+    meansOfTransportCrossingTheBorderIDNumber: String = "1",
```

Comment:
The same here

Reply:
Could not be empty - it is now mandatory field